store/models/job_spec.go

```diff
@@ -166,21 +178,23 @@ const (
 // Initiators will have their own unique ID, but will be associated
 // to a parent JobID.
 type Initiator struct {
-	ID    int    `json:"id" storm:"id,increment"`
-	JobID string `json:"jobId" storm:"index"`
+	ID        string `json:"id" gorm:"primary_key;not null"`
```

Comment:
Do initiators need a string ID? Seems like a uint would suffice, and then the database could assign the IDs instead of having to do it ourselves.

Reply:
👍 
I'm ok with a uint ID. I'll add it to the list.